src/spectrecoff-cli/commands/Output.fs

```diff
@@ -40,14 +45,17 @@ type OutputExample() =
         E "E is a short for Edgy." |> toConsole
         NL |> toConsole
 
-        // Using vanilla and composite styles
-        Vanilla "This let's you pass in a style as defined by Spectre." |> toConsole
+        // Using Vanila, Identity and composite styles
+        Vanilla "This let's you pass through without providing any styling." |> toConsole
         V "V is a short for Vanilla." |> toConsole
-        V "In order to be able to write the styles easily, there are some functions:" |> toConsole
-        V $"""You can use {markupString (Some Color.Purple) [ Decoration.Bold ] "the markup"} function,""" |> toConsole
-        V $"""or {calm "the calm"}, {pumped "the pumped"} {edgy "or the edgy"} functions""" |> toConsole
-        V "to utilize the same styles as defined in the current theme." |> toConsole
-        V $"""As you can see, {pumped "Vanilla"} is especially useful for styles in {edgy "one line!"} (more on that below).""" |> toConsole
+        V "Vanilla will ignore any markup in the string - it is escaped." |> toConsole
```

Comment:
Maybe call it `Escaped` then, so a caller knows what to expect? I don't think Vanilla really communicates that without reading this example or the code.

Reply:
True, but _all outputs are escaped_, that's the default. `Vanilla `is the one that _only escapes_, but does no styling (in contrast to, e.g., `Calm`, which escapes _and_ applies calm styles). So I think `Escaped `is also not a good name.

Reply:
Hmm, ok, I will just put `raw` and `plain` here then and let you decide if you think one fits better. But to twist the knive one last time: line 48 has a typo `Vanila` in it!

Reply:
The knive ... ?? Ah, you mean knife!

But in all seriousness, `Vanilla `was the name that existed for a long time, and I explained the need to introduce a new one. So for me, the discussion should center around the name of the _new type_, and not introduce any breaking changes. I think `Raw` is a good suggestion though, could roll with that.

Reply:
Argh, right back at me :( That will teach me in the future to stab knives at people.

Reply:
KK renamed to Raw, fixed the typo.